### PR TITLE
Note that all creds are encrypted.

### PR DESCRIPTION
**Pull request by matthewfischer**
All credentials are encrypted, regardless of type. State this clearly to avoid user confusion.

### CODE REVIEW

**Comment by crawsible** [comment]
I would reword this for clarity and accuracy. Something like:

> For every credential type, secret values are encrypted before storage. For instance, the private key of a certificate-type credential and the password of a user-type credential are encrypted before storage. For JSON and Value type credentials, the full contents are encrypted before storage.

**Comment by seviet** [comment]
thanks @matthewfischer! I see you're a Pivotal employee and am merging -- just a reminder to publicize your membership with Pivotal so gitbot knows you've signed the CLA.
